op-node/flags/flags.go

```diff
@@ -44,23 +44,24 @@ var (
 		Usage:   fmt.Sprintf("Predefined network selection. Available networks: %s", strings.Join(chaincfg.AvailableNetworks(), ", ")),
 		EnvVars: prefixEnvVars("NETWORK"),
 	}
+	/* Optional Flags */
 	RPCListenAddr = &cli.StringFlag{
 		Name:    "rpc.addr",
 		Usage:   "RPC listening address",
 		EnvVars: prefixEnvVars("RPC_ADDR"),
+		Value:   "0.0.0.0",
```

Comment:
We should not default to exposing the RPC port publicly.  Defaulting to `127.0.0.1` would be the safe choice.

Reply:
There's a couple other places we need to update stuff then.